table-api/src/main/java/io/deephaven/api/ColumnAssignment.java

```diff
@@ -0,0 +1,32 @@
+package io.deephaven.api;
+
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Parameter;
+
+@Immutable(builder = false, copy = false)
+public abstract class ColumnAssignment implements JoinAddition {
```

Comment:
Also a `Selectable`?

Reply:
Yes. Similar to `Filter`, about parsability of RHS.